src/main/java/svnserver/ext/gitea/mapping/GiteaAccess.java

```diff
@@ -75,6 +75,7 @@ public Repository load(@NotNull String userName) throws Exception {
                   repository.getPermissions().setPush(false);
                   return repository;
                 }
+                throw new FileNotFoundException();
```

Comment:
I don't understand the logic of this code. Doesn't your change mean that anonymous users cannot access any non-private repos?